server/neptune/gateway/event/check_suite_handler.go

```diff
@@ -40,9 +40,7 @@ func (h *CheckSuiteHandler) Handle(ctx context.Context, event CheckSuite) error
 
 func (h *CheckSuiteHandler) handle(ctx context.Context, event CheckSuite) error {
 	builderOptions := BuilderOptions{
```

Comment:
we don't filter on the app name or anything here, can we do that to limit when we're cloning potentially?

Reply:
I guess we only get these for our app

Reply:
Oh nvm this only gets called for re-requested so we should be fine i guess.